k8s-daemonset/helloworld/world.py

```diff
@@ -4,10 +4,11 @@
 app = Flask(__name__)
 
 pod_ip = os.getenv("POD_IP")
+target_world = os.getenv("TARGET_WORLD", "world")
```

Comment:
I think I'm missing something.  Should we be reading the world.txt file somewhere?